src/tagmanager/tm_parser.c

```diff
@@ -446,13 +446,14 @@ static TMParserMapEntry map_OBJC[] = {
 	{'M', tm_tag_macro_t},
 };
 
-/* not in universal-ctags */
 static TMParserMapEntry map_ASCIIDOC[] = {
-	{'n', tm_tag_namespace_t},
-	{'m', tm_tag_member_t},
-	{'d', tm_tag_macro_t},
-	{'v', tm_tag_variable_t},
-	{'s', tm_tag_struct_t},
+	{'c', tm_tag_namespace_t},
+	{'s', tm_tag_member_t},
+	{'S', tm_tag_macro_t},
+	{'t', tm_tag_variable_t},
+	{'T', tm_tag_struct_t},
+	{'u', tm_tag_undef_t},
+	{'a', tm_tag_undef_t},
```

Comment:
@techee the last two don't work, is that because you used `tm_undef_t`, and if so why not allow them to work?

Reply:
Ok, leave them for now, but they actually worked for me when I mapped them to two random things, though I have deleted that experiment, so I don't know what I used.